src/main/java/utils/ConditionUtils.java

```diff
@@ -0,0 +1,27 @@
+package utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ConditionUtils {
```

Comment:
```suggestion
public final class ConditionUtils {
    private ConditionUtils() {}
```

저는 static 메서드만을 모아두는 Util 클래스의 경우에는
혹시 실수로라도 Util 인스턴스가 생성되지 않도록,
생성자는 private 으로, class 는 final 로 선언하는 것을 좋아합니다.
취향이므로 꼭 반영해주시지 않아도 괜찮습니다.